### PR TITLE
Allow editing ingress privacy for private hypershift cluster

### PR DESCRIPTION
**Pull request by zgalor**
Related: [SDA-8109](https://issues.redhat.com//browse/SDA-8109)

### CODE REVIEW

File: cmd/edit/ingress/cmd.go [hunk 1]
```diff
@@ -118,10 +118,15 @@ func run(cmd *cobra.Command, argv []string) {
 		interactive.Enable()
 	}
 
+	cluster := r.FetchCluster()
 	var labelMatch *string
 	if cmd.Flags().Changed(labelMatchFlag) {
+		if ocm.IsHyperShiftCluster(cluster) {
+			r.Reporter.Errorf("Updating route selectors is not supported for hosted cp clusters")
```
Comment (gdbranco) [inline cmd/edit/ingress/cmd.go:125]: Is this already done in clusters service as well?

Comment (zgalor) [reply cmd/edit/ingress/cmd.go:125]: No. Just in rosa

Comment (zgalor) [reply cmd/edit/ingress/cmd.go:125]: I was wrong. It is done indirectly:

```
	if patch.RouteSelectors != nil {
		if ingress.Default {
			err = errors.BadRequest.UserErrorf(
				"Can't add route selectors to default ingress '%s' in cluster '%s'",
				ingress.ID,
				cluster.ID)
			return
		}
		clauses["route_selectors"] = patch.RouteSelectors
	}
```

**Pull request merged**